packages/wdio-allure-reporter/tests/allureFeatures.test.js

```diff
@@ -377,6 +377,29 @@ describe('auxiliary methods', () => {
         expect(addAttachment).toHaveBeenCalledTimes(1)
         expect(addAttachment).toHaveBeenCalledWith('foo', JSON.stringify(json, null, 2), 'application/json')
     })
+
+    it('should populate the correct deviceName', () => {
+        const capabilities = {
+            deviceName: 'emulator',
+            desired: {
+                platformName: 'Android',
+                automationName: 'UiAutomator2',
+                deviceName: 'Android GoogleAPI Emulator',
+                platformVersion: '6.0',
+                noReset: true,
+            }
+        }
+        const reporter = new AllureReporter({ stdout: true })
+        let addParameter = jest.fn()
+        let currentTestMock = { addParameter, addLabel: jest.fn() }
```

Comment:
no need to define it in an extra variable, also we should use `const` if we decide not to change the variable value
```suggestion
        const currentTestMock = {
            addParameter: jest.fn(),
            addLabel: jest.fn()
        }
```